internal/v2/cert_verifier/cert_verifier_test.go

```diff
@@ -35,39 +80,74 @@ func TestVerifyClientCertChain(t *testing.T) {
 	for _, tc := range []struct {
 		description              string
 		pool                     *x509.CertPool
-		expectedCommonName       string
 		rawCerts                 [][]byte
 		verifiedChains           [][]*x509.Certificate
-		err                      error
+		err			 error
 	}{
 		{
 			description: "static : empty certPool",
 			pool: certPool0,
-			expectedCommonName: "s2a_test_cert",
 			rawCerts: nil,
 			verifiedChains: nil,
-			err: nil,
+			err: errors.New("rpc error: code = Unknown desc = Client Peer Verification failed: client cert chain is empty."),
 		},
 		{
 			description: "static : non-empty certPool",
 			pool: certPool1,
-			expectedCommonName: "s2a_test_cert",
-			rawCerts: cert1.Certificate,
+			rawCerts: [][]byte{clientDERCert,},
 			verifiedChains: [][]*x509.Certificate{{cert1.Leaf},{}},
 			err: nil,
 		},
 	}{
 		t.Run(tc.description, func(t *testing.T) {
-			VerifyPeerCertificateFunc := VerifyClientCertificateChain(tc.expectedCommonName, tc.pool)
-			if got, want := VerifyPeerCertificateFunc(tc.rawCerts, tc.verifiedChains), tc.err; got != want {
-				t.Errorf("Peer Certificate verification failed: %v", got)
+			log.Printf("----TEST CASE: Verify Client Cert Chain: %s----", tc.description)
```

Comment:
IIRC, I don't think this is required since on failure t.Run will automatically tell which test case failed/succeeded.

Reply:
Will do! I left this in from when I was testing, made it easier to go through the logs